source/common/runtime/runtime_features.cc

```diff
@@ -78,6 +78,7 @@ constexpr const char* runtime_features[] = {
     "envoy.reloadable_features.listener_in_place_filterchain_update",
     "envoy.reloadable_features.new_tcp_connection_pool",
     "envoy.reloadable_features.overload_manager_disable_keepalive_drain_http2",
+    "envoy.reloadable_features.prefer_quic_kernel_bpf_packet_routing",
```

Comment:
Do we need feature flags for alpha/incomplete features? Seems like if we want a change here we can just make it?

Reply:
I think I see now that this is to prevent accidently using this feature when it's slow? We don't have any user docs yet for QUIC/HTTP/3, but can we track this for documentation? I think I opened a docs issue for QUIC?

Reply:
Actually, I added it so I could force tests to use the "other" mode. And I figured there might be some case where someone might need to force this, such as a bug in the BPF program that isn't present in the user-mode-software version.

But I could take it out, and make the tests get the right behavior through some other method easily enough.

Reply:
> But I could take it out, and make the tests get the right behavior through some other method easily enough.

+1

Reply:
Correction: it's not that easy to do this, because it's in an integration test. I think I'd need to add proto config (probably AUTO, USE_KERNEL, USE_USERSPACE or similar). Is that preferable to the runtime override?

Reply:
Runtime is fine, but if we put it in this list we should probably end up documenting it.

Reply:
Huh, there's currently no appropriate place to put docs for this. I'll add a note in #12923 to document it.

Reply:
Yup that's perfect, thanks.

Reply:
Added.

Reply:
The easiest way is to put it in the bootstrap config. Example here: https://github.com/envoyproxy/envoy/blob/1a54bbb0a6fb2dec03cddea63ba80bb198180664/configs/using_deprecated_config.v2.yaml#L66